src/components/NonUserMainComponent.js

```diff
@@ -4,7 +4,8 @@ import white_cloud from "../../assets/image/NonUserMain/white-cloud.png";
 import search from "../../assets/image/NonUserMain/search.png";
 import user from "../../assets/image/NonUserMain/user.png";
 import artist from "../../assets/image/NonUserMain/artist.png";
```

Comment:
```suggestion
import search from "../assets/image/NonUserMain/search.png";
import user from "../assets/image/NonUserMain/user.png";
import artist from "../assets/image/NonUserMain/artist.png";
```
white_cloud 경로도 수정 부탁드립니다. 코드 수정 요청이 4번째 줄 부터 가능해서 선택을 못했습니다ㅠ